tests/integration/test_rootcheck/test_configuration/test_rootcheck.py

```diff
@@ -0,0 +1,253 @@
+'''
+copyright: Copyright (C) 2015-2023, Wazuh Inc.
+           Created by Wazuh, Inc. <info@wazuh.com>.
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: The 'rootcheck' tool allows to define policies in order to check if the agents
+       meet the requirement specified. The rootcheck engine can check if a process is running, if a file is
+       present and if the content of a file contains a pattern,
+       or if a Windows registry key contains a string or is simply present.
+
+components:
+    - rootcheck
+
+targets:
+    - manager
+
+daemons:
+    - wazuh-analysisd
+
+os_platform:
+    - linux
+
+os_version:
+    - Arch Linux
+    - Amazon Linux 2
+    - Amazon Linux 1
+    - CentOS 8
+    - CentOS 7
+    - Debian Buster
+    - Red Hat 8
+    - Ubuntu Focal
+    - Ubuntu Bionic
+
+references:
+    - https://documentation.wazuh.com/current/user-manual/capabilities/policy-monitoring/rootcheck
+    - https://documentation.wazuh.com/current/user-manual/reference/daemons/wazuh-analysisd.html
+
+tags:
+    - rootcheck
+'''
+
+import json
+import os
+import time
+import pytest
+from pathlib import Path
+
+from wazuh_testing.constants.paths.logs import ALERTS_JSON_PATH
+from wazuh_testing.constants.paths.sockets import QUEUE_DB_PATH, WAZUH_DB_SOCKET_PATH
+from wazuh_testing.tools.db_administrator import DatabaseAdministrator
+from wazuh_testing.tools.simulators.agent_simulator import Sender, Injector, create_agents, connect
+from wazuh_testing.tools.socket_controller import SocketController
+from wazuh_testing.utils import configuration
+from wazuh_testing.utils.network import TCP
+from wazuh_testing.utils.services import control_service
+from wazuh_testing.utils.manage_agents import remove_agents
+
+
+from . import CONFIGS_PATH, TEST_CASES_PATH
+
+# Marks
+pytestmark = [pytest.mark.linux, pytest.mark.tier(level=0), pytest.mark.server]
+injectors = []
+agents = []
```

Comment:
This could be removed, but for now, I have to continue debugging. But I just need to return the agent.